transport/http/outbound.go

```diff
@@ -256,6 +256,12 @@ func (o *Outbound) call(ctx context.Context, treq *transport.Request) (*transpor
 
 	span.SetTag("http.status_code", response.StatusCode)
 
+	// Service name match validation, return yarpcerrors.CodeInternal error if not match
+	if match, resSvcName := checkServiceMatch(treq.Service, &response.Header); !match {
```

Comment:
a map is just a pointer, so `&map` seems unnecessary here

Reply:
Resolved. Interface and invoke statement are updated.